### ISSUE TITLE
evil-ediff removed from melpa

### ISSUE THREAD

**Issue by Eilie** [syl20bnr/spacemacs#10972]
It's gone, https://melpa.org/#/?q=evil-ediff . They say it has been replaced by evil-collection.

**Comment by Compro-Prasad** [comment]
I don't see spacemacs uses evil-collection.

**Comment by Compro-Prasad** [comment]
Yup but evil-ediff has moved to evil-collection which isn't in spacemacs by default which is the problem.

**Comment by jjzmajic** [comment]
@thinkinnight, same. Just bumping it's more evident to people facing the same issue that it is indeed a Spacemacs problem and not a user-config problem. Hopefully, it will get resolved soon.

**Comment by jsparedes** [comment]
The same error message:

> Package evil-ediff is unavailable. Is the package name misspelled?

**Comment by JamesHeshka** [comment]
Temp fix - haven't checked into changes for evil-collection. 

You can remove package requirement/error by remove line 14: evil-ediff from 
~/.emacs.d/layers/+spacemacs/spacemacs-evil/packages.el

**Comment by abellmann** [comment]
evil-ediff is gone from melpa again...

My workaround in .spacemacs/dotspacemacs-additional-packages

````
     ;; evil-ediff was removed from elpa, but is still referenced in spacemacs-edit layer...
     (evil-ediff :location (recipe :fetcher github :repo "emacs-evil/evil-ediff"))
````
